src/common/log.cpp

```diff
@@ -1121,6 +1125,12 @@ static const wxChar* GetSysErrorMsg(wxChar* szBuf, size_t sizeBuf, unsigned long
         errorMsg = strerror_r((int)nErrCode, buffer, sizeof(buffer));
 #elif defined( __VMS )
         errorMsg = strerror((int)nErrCode);
+#elif defined(__WXOSX_COCOA__)
+        int iErrorCode = (int)nErrCode;
+        if ( iErrorCode < 0 )
+            strncpy(buffer,GetMacOSStatusErrorString(iErrorCode), sizeof(buffer));
```

Comment:
This function is deprecated since 10.8, is there really no replacement (something in `NSError`?) that we could use instead?

Reply:
unfortunately not ... at least not something that is as precise

https://stackoverflow.com/questions/14699604/replacements-for-getmacosstatuserrorstring-getmacosstatuscommentstring

and the fact that the error strings are not useful is still true in 10.15 ...